app/workers/mhv/audit_logout_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module MHV
+  class AuditLogoutJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: false
+
+    def perform(uuid, mhv_correlation_id)
+      MHVLogging::Client.new(session: { user_id: mhv_correlation_id })
```

Comment:
I would possibly consider first looking up the user, and 

`return false unless user.mhv_last_signed_in` but I could see argument against as well.